lib/executor/ramping_arrival_rate_test.go

```diff
@@ -713,7 +713,7 @@ func TestRampingArrivalRateGlobalIters(t *testing.T) {
 		BaseConfig:      BaseConfig{GracefulStop: types.NullDurationFrom(100 * time.Millisecond)},
 		TimeUnit:        types.NullDurationFrom(950 * time.Millisecond),
 		StartRate:       null.IntFrom(0),
-		PreAllocatedVUs: null.IntFrom(2),
+		PreAllocatedVUs: null.IntFrom(4),
```

Comment:
Is this needed ? we should be spreading over the maxVUs either way :thinking:

Reply:
it is needed, otherwise the last segment will get 0 pre-initialized VUs and 1 max VUs, and will drop all of its iterations while it's trying to initialize that VU.

Reply:
As you say, the (re-)segmentation is based on MaxVUs, so nothing changes in the actual iteration numbers, but we actually also need a VU to run the planned iterations :sweat_smile: